_rules/bd5de3.md

```diff
@@ -0,0 +1,37 @@
+
+---
+name: Focusable element has visible focus
+test_type: atomic
+
+ description: |
+  This rule checks that all elements that can receive focus has a focus styling that is visibly different from the normal styling.
+ success_criterion: 
+- 2.4.7 # Focus Visible
+
+ test_aspects:
+- DOM Tree
+- CSS Styling
+
+ authors:
+- Anne Thyme Nørregaard
+---
+
+## Applicability
+
+This rule applies to all elements that can receive focus.
```

Comment:
This needs to specify somehow that elements with a tabindex="-1" which can still receive focus with the help of javascript are still applicable. I am unsure whether or not such an applicability can be tested automatically.